programmers2/실패율.js

```diff
@@ -0,0 +1,13 @@
+function solution(N, stages) {
+    const arr = Array.from(Array(N), () => new Array(2));
+    let allPlayer = stages.length;
+    for (let i = 1; i < N + 1; i++) {
+        const nowStepPlayer = stages.filter(a => a === i).length;
```

Comment:
```suggestion
        const nowStepPlayer = stages.filter((element) => element === i).length;
```
arrow function parameter 부분 항상 괄호 치기
a보다 의미있는 변수명 지정